vmdb/app/assets/javascripts/miq_application.js

```diff
@@ -1347,7 +1356,7 @@ function miqSetLayoutHeader(unitId, text) {
 
 // common function to pass ajax request to server
 function miqAjaxRequest(itemId, path) {
-  if (miqCheckForChanges() == false) {
+  if (!miqCheckForChanges()) {
```

Comment:
check the return value of `miqCheckForChanges()` to make sure this is correct otherwise == false is different to !